light-curve/light_curve/light_curve_py/features/roms.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+
+from ._base import BaseSingleBandFeature
+
+
+class Roms(BaseSingleBandFeature):
+    r"""Robust median statistic
+
+    RoMS = $\[ (N-1)^{-1} \sum_{i=1}^{N} \frac{|m_i - median(m_i)|}{\sigma_i} \]$
```

Comment:
Let's change it to `$$ equation $$` style, like in Rust features, see an example [here](https://github.com/light-curve/light-curve-feature/blob/b159861d14d4fb1656a7aa89bf8048006e915f9f/src/features/anderson_darling_normal.rs#L10-L12)

```suggestion
    $$
    \frac1{N-1} \sum_{i=0}^{N-1} \frac{|m_i - \mathrm{median}(m_i)|}{\sigma_i}
    $$
```